README.md

```diff
@@ -1,13 +1,14 @@
 
 # Anvil - reactive views for Android
 
-[![Join the chat at https://gitter.im/zserge/anvil](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/zserge/anvil?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Build Status](https://travis-ci.org/zserge/anvil.svg?branch=master)](https://travis-ci.org/zserge/anvil)
+[![Join Telegram chat at @inkremental](https://img.shields.io/badge/inkremental-Join%20chat-2CA5E0?logo=telegram)](https://t.me/inkremental)
```

Comment:
Since current channel consist primarily from  Russian speaking members, this can make negative effect for attracting future contributors. How about making also english speaking Telegram channel?

Reply:
Channel on kotlinlang Slack might be more preferable for this, I guess, since English-speaking kotliners are mostly there.

Reply:
Created #inkremental, will link it here tomorrow.